### ISSUE TITLE
Count editing on github.com as a separate IDE

### ISSUE THREAD

**Issue by Matt-Gleich** [wakatime/chrome-wakatime#81]
If you do this please make it so that you can have the chrome extension either not count it separate or to ignore all other sites and only with github.com.

**Comment by alanhamlett** [comment]
Do you mean github.dev, GitHub's vscode in the browser, or editing and committing a document in the github.com text editor?

For github.dev, the vscode plugin works there if you install it from the extensions menu.

Here's a related issue about detecting the project from the url: #47

**Comment by alanhamlett** [comment]
We should definitely detect the project from the github repo url, similar to #180 and #181.

**Comment by alanhamlett** [comment]
Yes, let's try adding those fields when editing files on `github.com` and when using the web vscode editor on `github.dev`.

Normal browsing files on `github.com` without editing a file will only send the project, not editor nor language, same as it currently does.

**Comment by alanhamlett** [comment]
Related: #82.
